lib/buffer.js

```diff
@@ -1024,6 +1024,13 @@ function checkOffset(offset, ext, length) {
     throw new errors.RangeError('ERR_INDEX_OUT_OF_RANGE');
 }
 
+function checkByteLength(byteLength) {
+  if (byteLength < 1 || byteLength > 6)
```

Comment:
Can you add curly braces to this `if` statement since the body spans multiple lines.

Reply:
Fixed this one and a few other similar instances in the same file.